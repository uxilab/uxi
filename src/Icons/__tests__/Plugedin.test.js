import React from 'react';
import Plugedin from '../Plugedin';

describe('Icon : <Plugedin />', () => {
  it('match snapshot', () => {
    expect(shallow(<Plugedin />)).toMatchSnapshot();
  });
});

