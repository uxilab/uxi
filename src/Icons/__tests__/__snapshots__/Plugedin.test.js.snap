// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon : <Plugedin /> match snapshot 1`] = `
<SvgIcon
  color="currentColor"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  size={24}
  style={Object {}}
  viewBox="0 0 24 24"
>
  <svg
    focusable="false"
    height="24px"
    viewBox="0 0 16.1 16.65"
    width="24px"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g
      data-name="Layer 2"
      id="Layer_2"
    >
      <g
        data-name="Layer 1"
        id="Layer_1-2"
      >
        <path
          d="M16.1,0H0V1.3H2.05V3.78a6,6,0,0,0,5.35,6v2.1a1.71,1.71,0,0,0,1.71,1.7.41.41,0,0,1,.4.41v2.7h1.3V14a1.71,1.71,0,0,0-1.7-1.71.4.4,0,0,1-.41-.4V9.74a6,6,0,0,0,5.35-6V1.3H16.1ZM12.75,3.78a4.7,4.7,0,0,1-9.4,0V1.3h9.4Z"
        />
      </g>
    </g>
  </svg>
</SvgIcon>
`;
